frontend/src/App.jsx

```diff
@@ -109,6 +113,8 @@ const App = () => {
   };
 
   return (
+
```

Comment:
![Screenshot 2024-05-31 110352](https://github.com/Satyam1923/Spring-Music-Player/assets/104068640/40ed3b1b-5698-4684-b321-9af22a6f894a)
why removed rest of the code.

Reply:
I just added a top div so i can add my footer component thats it